arrProblem.js

```diff
@@ -0,0 +1,30 @@
+const array = [
+    { id: 'one', next: 'two', value: null },
+    { id: 'two', next: 'three', value: null },
+    { id: 'three', next: null, value: null },
+  ];
+
+function arrProb(arr){
+    elements =[]

```

Comment:
always try to go for immutable data structures. Like if we needed to return an array we could've used map here